pull/context.go

```diff
@@ -92,6 +92,9 @@ type Context interface {
 	// implementation dependent.
 	Reviews() ([]*Review, error)
 
+	// DismissPullRequestReview dismisses a review on a Pull Request.
+	DismissPullRequestReview(reviewID string, message string) error
```

Comment:
This should be defined in the `handler` package where it is used. The `pull.Context` type is used for reading information about pull requests and by convention, all mutations happen directly in the handlers (assigning reviewers, posting status checks, etc.)